src/Powercord/coremods/moduleManager/index.js

```diff
@@ -0,0 +1,290 @@
+const { loadStyle } = require('../util');
+
+const { existsSync } = require('fs');
+const { writeFile, readFile } = require('fs').promises;
+const { React, getModule, i18n: { Messages } } = require('powercord/webpack');
+const { PopoutWindow } = require('powercord/components');
+const { inject, uninject } = require('powercord/injector');
+const { findInReactTree, forceUpdateElement } = require('powercord/util');
+const { SpecialChannels: { CSS_SNIPPETS, STORE_PLUGINS, STORE_THEMES }, WEBSITE } = require('powercord/constants');
+const { join } = require('path');
+const commands = require('./commands');
+const deeplinks = require('./deeplinks');
+const i18n = require('./licenses/index');
+
+const Plugins = require('./components/manage/Plugins');
+const Themes = require('./components/manage/Themes');
+const QuickCSS = require('./components/manage/QuickCSS');
+const SnippetButton = require('./components/SnippetButton');
+const InstallerButton = require('./components/installer/Button');
+const { cloneRepo, getRepoInfo } = require('./util');
+const { injectContextMenu } = require('powercord/util');
+
+const Menu = getModule([ 'MenuItem' ], false);
+
+let _quickCSS = '';
+const _quickCSSFile = join(__dirname, 'quickcss.css');
```

Comment:
`quickcss.css` still exists under `pc-moduleManager` under plugins dir. Just make it watch for that file and copy it over to the coremod and delete `pc-moduleManager`

Reply:
Set up a migration script, please test it.